core/src/test/scala/io/projectglow/sql/util/ExpectsGenotypeFieldsSuite.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2019 The Glow Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.projectglow.sql.util
+
+import org.apache.spark.sql.AnalysisException
+import org.apache.spark.sql.functions._
+
+import io.projectglow.Glow
+import io.projectglow.functions._
+import io.projectglow.sql.GlowBaseTest
+
+class ExpectsGenotypeFieldsSuite extends GlowBaseTest {
+  lazy val gatkTestVcf =
+    s"$testDataHome/variantsplitternormalizer-test/test_left_align_hg38_altered.vcf"
+  lazy val sess = spark
+
+  // This is how we originally detected an issue where ExpectsGenotypeFields succeeds during
```

Comment:
For documentation, it may be useful to add a PR link to this

Reply:
What's the PR?

Reply:
Oh, I meant it'd be good to link or tag this PR just for posterity. I don't mind either way.

Reply:
Ah got it. Done.